File: alurapic/src/app/core/token/api-token.ts
export interface ApiToken
{
    token: string;
    refreshToken: string;
}

export class ApiToken
{
    token: string;
    refreshToken: string;

    constructor (token: string, refreshToken: string) {
        this.token = token;
        this.refreshToken = refreshToken; 
    }
}

File: api/Models/ApiToken.cs
using System;
using Microsoft.AspNetCore.Authentication.JwtBearer;

namespace api.Models
{
    public class ApiToken
    {
        public ApiToken(string token, string refreshToken)
        {
            Token = token;
            RefreshToken = refreshToken;
        }

        public string Token {get;}
        public string RefreshToken {get;}
    }
}